Library/Formula/jump.rb

```diff
@@ -0,0 +1,36 @@
+require "language/go"
+
+class Jump < Formula
+  desc "A quick and fuzzy directory jumper."
+  homepage "https://github.com/gsamokovarov/jump"
+  url "https://codeload.github.com/gsamokovarov/jump/tar.gz/v0.4.0"
+  sha256 "d105eebf832c6792ea72eaede2e0b7a14bb9da1a9e2b27847a6954c3bb9071d0"
+  head "https://github.com/gsamokovarov/jump.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    mkdir_p buildpath/"src/github.com/gsamokovarov"
+    ln_s buildpath, buildpath/"src/github.com/gsamokovarov/jump"
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    system "go build -o jump"
+    bin.install "jump"
+    man1.install "man/jump.1"
+  end
+
+  def caveats; <<-EOS.undent
+    To install shell integration, follow the instructions at:
+      #{prefix}/bin/jump shell
+    EOS
+  end
```

Comment:
Thanks for the hint! I think we should keep this line (or some form of it) as you gotta integrate it with your shell to make it usable. E.g. get the fancy `j` shell function.